src/context.cpp

```diff
@@ -85,23 +85,29 @@ bool contains(const std::shared_ptr<librealsense::device_info>& first,
 
 namespace librealsense
 {
-    std::map<uint32_t, rs2_format> platform_color_fourcc_to_rs2_format = {
+    const std::map<uint32_t, rs2_format> platform_color_fourcc_to_rs2_format = {
         {rs_fourcc('Y','U','Y','2'), RS2_FORMAT_YUYV},
         {rs_fourcc('U','Y','V','Y'), RS2_FORMAT_UYVY}
     };
-    std::map<uint32_t, rs2_stream> platform_color_fourcc_to_rs2_stream = {
+    const std::map<uint32_t, rs2_stream> platform_color_fourcc_to_rs2_stream = {
         {rs_fourcc('Y','U','Y','2'), RS2_STREAM_COLOR},
         {rs_fourcc('U','Y','V','Y'), RS2_STREAM_COLOR}
     };
 
+    bool context::one_time_msg = false;
```

Comment:
If this is only for the context ctor, then maybe it'd be simpler to just make it static inside the ctor... that way, no need to refer to it in multiple places. Up to you.

If you do keep it, I'd suggest another name, 'version_message_logged' or similar.

Reply:
Done 👍